src/main/scala/com/cognite/spark/datasource/EventsRelation.scala

```diff
@@ -48,7 +48,7 @@ object UpdateEventItem {
       Setter[String](eventItem.`type`),
       Setter[String](eventItem.subtype),
       Setter[Map[String, String]](eventItem.metadata),
-      Setter[Seq[Long]](eventItem.assetIds),
+      eventItem.assetIds.map(a => Map("set" -> a)),
```

Comment:
Interesting, I see we already have this for `securityCategories` in time series metadata as well.